app/src/main/java/com/eveningoutpost/dexdrip/Models/BgReading.java

```diff
@@ -709,9 +710,9 @@ public void find_new_curve() {
         }
     }
 
-    public void calculateAgeAdjustedRawValue(){
+    public void calculateAgeAdjustedRawValue(Context context){
```

Comment:
Single point of return is a good thing when it is combined with another keyword. Since this word is not used in java, it is a bad thing. Not specifying the keyword but you can see it here (www.javaworld.com/community/node/3375). Specifically picked a link that does not specify the word itself.